_layouts/formula.html

```diff
@@ -114,14 +119,20 @@ <h2>{{ f.name }}</h2>
 </table>
 {%- endif -%}
 
-{%- if site.data.analytics.install.homebrew-core["30d"].formulae[full_name].size > 0 -%}
+{%- if formula_path == "formula-linux" -%}
+{%- assign analytics_data_source = "linuxbrew-core" -%}
+{%- else -%}
+{%- assign analytics_data_source = "homebrew-core" -%}
+{%- endif -%}
+
+{%- if site.data.analytics.install[analytics_data_source]["30d"].formulae[full_name].size > 0 -%}
 <p>Analytics:</p>
 <table>
     <tr>
         <th colspan="2">Installs (30 days)</th>
     </tr>
 
-    {%- for fa in site.data.analytics.install.homebrew-core["30d"].formulae[full_name] -%}
+    {%- for fa in site.data.analytics.install[analytics_data_source]["30d"].formulae[full_name] -%}
```

Comment:
This works when `homebrew-core` but not `linuxbrew-core` because the files don’t exist in `_site`?

Reply:
Maybe just skip this for `linuxbrew-core`? I think this is some special-cased code in `brew formula-analytics`.

Reply:
Why does it exist as a special case? What does it do differently to the non-`homebrew-core` scoped ones? If we skip these (all the ones where I've changed `homebrew-core` to `analytics_data_source`), then we'll not have any analytics data on the formula pages?

Reply:
(Let's just say I understand the formulae side of this better than the analytics side, in fact I didn't even realise the analytics pages were a thing until I noticed the numbers were suspiciously similar between macOS and Linux pages. :smile:)

Reply:
Check out https://github.com/Homebrew/homebrew-formula-analytics/blob/fdada48d6353915fa852d6a6af11a4280afc1df2/cmd/brew-formula-analytics.rb#L120-L132

We only have formulae pages for homebrew-core formulae and want to display their analytics data inline. This is the way of doing this that doesn't involve calling `brew formula-analytics` thousands of times for each formula.

Reply:
> If we skip these (all the ones where I've changed `homebrew-core` to `analytics_data_source`), then we'll not have any analytics data on the formula pages?

Yes.

I'd suggest testing this out locally and seeing what it looks like with your generated data there.

Reply:
I made it work rather than removing it.

![Screenshot from 2019-10-18 15-14-46](https://user-images.githubusercontent.com/355033/67101776-2d6f7100-f1ba-11e9-8183-c5e17bbaeead.png)